image-rs/src/verity.rs

```diff
@@ -0,0 +1,102 @@
+use anyhow::Result;
+use base64;
+use devicemapper::{DevId, DmFlags, DmName, DmOptions, DM};
+use serde::{Deserialize, Serialize};
+use serde_json;
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct DmVerityOption {
+    /// Hash algorithm for dm-verity.
+    pub hashtype: String,
+    /// Used block size for the data device.
+    pub blocksize: String,
```

Comment:
how about using `u64` instead of `String`?

Reply:
Thanks, I'll update it.